tool_commands/command_box.py

```diff
@@ -115,21 +129,20 @@ def run_script(self, qm_idx):
 
             module_path = self.get_module_path(display_name)
             if not module_path.is_file():
-                logger.eror(f"{module_path} is not a file")
+                logger.error(f"{module_path} is not a file")
```

Comment:
Hier maakt het niet uit, maar als je in django projecten a/h werk gaat moet je deze f-stijl niet gebruiken, maar ouderwets `%s` en een komma gebruiken. I.v.m. automatische sentry log message groepering.